docs/installation.md

```diff
@@ -7,11 +7,11 @@ VSCode.
 
 Download and install the VSCode extension.
 
-- For Arm MacOS: [publisher-1.1.5-darwin-arm64.vsix](https://cdn.posit.co/publisher/releases/tags/v1.1.5/publisher-1.1.5-darwin-arm64.vsix)
-- For Intel MacOS: [publisher-1.1.5-darwin-amd64.vsix](https://cdn.posit.co/publisher/releases/tags/v1.1.5/publisher-1.1.5-darwin-amd64.vsix)
-- For Windows: [publisher-1.1.5-windows-amd64.vsix](https://cdn.posit.co/publisher/releases/tags/v1.1.5/publisher-1.1.5-windows-amd64.vsix)
-- For Arm Linux: [publisher-1.1.5-linux-arm64.vsix](https://cdn.posit.co/publisher/releases/tags/v1.1.5/publisher-1.1.5-linux-arm64.vsix)
-- For Intel Linux: [publisher-1.1.5-linux-amd64.vsix](https://cdn.posit.co/publisher/releases/tags/v1.1.5/publisher-1.1.5-linux-amd64.vsix)
+- For Arm MacOS: [publisher-1.1.6-darwin-arm64.vsix](https://cdn.posit.co/publisher/releases/tags/v1.1.6/publisher-1.1.6-darwin-arm64.vsix)
```

Comment:
Shouldn't we just update this now to download from the marketplace?

Reply:
I added instructions and a link to the marketplace page for installation. I kept the manual installation instructions to keep this PR from becoming destructive, but I can absolutely remove them.